mods/cnc/maps/gdi03/map.yaml

```diff
@@ -823,15 +823,35 @@ Rules:
 			Amount: -10
 		-Capturable:
 	HQ:
+		Buildable:
+			Prerequisites: ~disabled
+	NOHQ:
+		Inherits: HQ
+		Buildable:
+			Prerequisites: proc
+		Tooltip:
+			Description: Provides an overview of the battlefield.\n  Requires power to operate.
+		-AirstrikePower:
+		RenderBuilding:
+			Image: hq
+	airstrike.proxy:
```

Comment:
I mean just disable the HQ build and don't give a substitute NOHQ, because otherwise the tech level is higher in mission 3 compared to mission 5 which is also inconsistent.